compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPUCheckIRBeforeLLVMConversion.cpp

```diff
@@ -45,6 +41,16 @@ checkStackAllocationSize(mlir::FunctionOpInterface funcOp) {
   if (funcOp.getFunctionBody().empty())
     return success();
 
+  unsigned maxAllocationSizeInBytes = 32768;
```

Comment:
The above function comment needs to be updated. We no longer have `clMaxAllocationSizeInBytes`, and we should document that what the default value is in the comment.

Reply:
Good point. I've updated the comment - could you please check if the current approach is fine with a clear reference to the default definition?